README.md

```diff
@@ -1,19 +1,28 @@
-# Ore Cart App
+# Ore Cart App [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+The master monorepo for all OreCart app software! The OreCart app includes a backend made with [FastAPI](https://fastapi.tiangolo.com) and a crossplatform mobile app made with [React-Native](https://reactnative.dev).
 
-For more details please see the Google Drive.
+For full documentation, please see our Google Drive (contributors can gain access by talking with the leads). The frontend and backend folders have READMEs with more information about developing in each as well.
```

Comment:
Maybe "project details" would be a better way to say "full documentation."

We want all the actual code documentation to be available to anyone who wants to view the repo.